week1/pages/add.js

```diff
@@ -0,0 +1,16 @@
+
+import HTTP from 'http'
+import STATUS  from './status.js'
+ 
+ 
+
+module.exports = {
+    "body" : function contentHomePage(Response ){
+        STATUS.varia = STATUS.varia + 1 ;
```

Comment:
I think it would be more maintainable if these page function didn't modify the state variable. You could pass the state variable as an extra argument. For example,

```javascript
function contentHomePage(response, state) {
  /** function body here */
}
```

What do you think? Does it make sense?